### PR TITLE
Build home page

### PR DESCRIPTION
**Pull request by JoseLopezSolis**
### Summary
This pull request introduces significant updates to our monkey hats e-commerce website. The changes include a refreshed home page design, enhanced product displays.

### Changes Made

1. **Home Page Styling**
   - Updated the home page with a playful and eye-catching jungle-inspired design.
   - Improved the overall layout for a more user-friendly experience.
   - Designed to create a visually appealing and engaging entry point for visitors.

2. **Product Display**
   - Curated a selection of our top-selling monkey hats for prominent display.
   - Highlighted key features of our products, including materials, sizes, and unique designs.
   - Emphasized the fun and warmth that our monkey hats bring to customers.

**Screenshots**
image
image

image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -7,11 +7,62 @@
...
+ }
+
+ .card-design img:hover{
+  transform: scale(1.2);
+  transition: transform 250ms cubic-bezier(0.165, 0.84, 0.44, 1);
+ }
+
+ .button-cart{
+  background-color: $black-background;
+  color: #fff;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss:35]: Define a variable for this color.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -7,11 +7,62 @@
...
+  
+  border: none;
+  width: 100%;
+  text-align: center;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss:39]: sort alphabetically this lines

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -7,11 +7,62 @@
...
+ }
+
+ .button-cart:hover{
+  background-color: $brand-color-yellow;
+  color: $black-background;
+ }
+
+ .button-design-products{
+  color: #000000;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss:48]: define a variable here too

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -7,11 +7,62 @@
...
+  background-color: transparent;
+  width: 200px;
+  border: none;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss:51]: sort it

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -7,11 +7,62 @@
...
+
+
+ @media (max-width: 767px) {
+  .grid-design{
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+   }
+}
+
+ // **END**//
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss:68]: add a blank line at the end of this file

File: app/views/devise/sessions/new.html.erb [hunk 6]
```diff
@@ -28,8 +28,14 @@
           <% unless params[:controller] == "devise/passwords" && params[:action] == "new" || params[:controller] == "devise/registrations" && params[:action] == "new" %>
             <button class="btn btn-lg btn-primary btn-login text-uppercase fw-bold mb-2" type="submit">Sign in</button>
           <% end %>
+
+            <div class="text-left mt-3">
+          <%- if controller_name != 'sessions' %>
+            <%= link_to "Log in", new_session_path(resource_name), class: "text-orange" %><br />
+          <% end %>
           <%= render "devise/shared/links" %>
         </div>
```
Comment (JuanequeX) [inline app/views/devise/sessions/new.html.erb:37]: delete the `<br>` and indented corrrectly

File: app/views/partials/Banners/_CoheteBanner.html.erb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'CoheteBanner.webp', class: 'img-fluid py-3', alt: 'CoheteBanner' %>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/partials/Banners/_CoheteBanner.html.erb:5]: indented correctly and add a blank line at the end

File: app/views/partials/Banners/_ElReyBanner.html.erb [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'ElReyBanner.webp', class: 'img-fluid py-3', alt: 'ElReyBanner' %>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/partials/Banners/_ElReyBanner.html.erb:5]: indented correctly and add a blank line at the end

File: app/views/partials/Banners/_JuniorHBanner.html.erb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'Junior.webp', class: 'img-fluid py-3', alt: 'Junior Image' %>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/partials/Banners/_JuniorHBanner.html.erb:5]: indented correctly and add a blank line at the end

File: app/views/partials/Banners/_ReyColoresBanner.html.erb [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'reyColoresBanner.webp', class: 'img-fluid py-3', alt: 'ReyColoresBanner' %>
+  </div>
+</div>
```
Comment (JuanequeX) [inline app/views/partials/Banners/_ReyColoresBanner.html.erb:5]: indented correctly and add a blank line at the end

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/devise/shared/_links.html.erb [hunk 11]
```diff
@@ -24,4 +19,4 @@
         <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), data: { turbo: false }, class: "text-dark" %><br />
       <% end %>
     <% end %>
-  </div>
+      <%= link_to "Go to home", home_index_path, class: "text-orange" %><br />
```
Comment (juperameza) [inline app/views/devise/shared/_links.html.erb:22]: Add a blank line at the end of this file

File: app/views/home/index.html.erb [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+<%= render partial: "partials/Products/products" %>
+<!---Section 4 -->
+<%= render partial: "partials/Banners/FoodMoney" %>
+<%= render partial: "partials/Products/products" %>
+<!---Section 5 -->
+<%= render partial: "partials/Banners/GranadaBanner" %>
+<%= render partial: "partials/Products/products" %>
+<!---Section 6 -->
+<%= render partial: "partials/Banners/ReyColoresBanner" %>
+<%= render partial: "partials/Products/products" %>
```
Comment (juperameza) [inline app/views/home/index.html.erb:18]: Add a blank line at the end of this file

File: app/views/partials/Banners/_FoodMoney.html.erb [hunk 13]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'FoodMoneyBanner.webp', class: 'img-fluid py-3', alt: 'FoodMoney' %>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/partials/Banners/_FoodMoney.html.erb:5]: Blank line

File: app/views/partials/Banners/_GranadaBanner.html.erb [hunk 14]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <div class="row justify-content-center">
+      <%= image_tag 'granadaBanner.webp', class: 'img-fluid py-3', alt: 'GranadaBanner' %>
+  </div>
+</div>
```
Comment (juperameza) [inline app/views/partials/Banners/_GranadaBanner.html.erb:5]: BLank line

File: app/views/partials/Products/_products.html.erb [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+          <a href="#" class="btn btn-primary button-cart">Add to Cart</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row justify-content-center mt-3 button-show-more my-3">
+      <button class="btn btn-dark button-design-products">Show more</button>
+    </div>
+</div>
```
Comment (juperameza) [inline app/views/partials/Products/_products.html.erb:97]: Add a blank line at the end here

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
lgtm

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**